apps/web/src/components/PoolStatusTable/PoolStatusTable.tsx

```diff
@@ -26,7 +26,9 @@ const PoolStatusTable = ({ user, pools }: UserInformationProps) => {
   const intl = useIntl();
   const paths = useRoutes();
 
-  if (isEmpty(user.poolCandidates)) {
+  const poolCandidatesArray = user.poolCandidates?.filter(notEmpty);
+
+  if (!poolCandidatesArray || poolCandidatesArray.length === 0) {
```

Comment:
Benefit of also type checking, ensuring it is not undefined when sorting below. Which the prior did not seem to do.